htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -825,11 +825,13 @@ function setforpredef() {
 		<?php if (empty($conf->global->MAIN_DISABLE_EDIT_PREDEF_PRICEHT)) { ?>
 			jQuery("#price_ht").val('').show();
 			jQuery("#multicurrency_price_ht").val('').show();
+			jQuery("#title_up_ht, #title_up_ht_currency").show();
 		<?php } else { ?>
 			jQuery("#price_ht").val('').hide();
 			jQuery("#multicurrency_price_ht").val('').hide();
+			jQuery("#title_up_ht, #title_up_ht_currency").hide();
 		<?php } ?>
-		jQuery("#price_ttc, #fourn_ref, #tva_tx, #title_vat, #title_up_ht_currency, #title_up_ttc, #title_up_ttc_currency").hide();
+		jQuery("#price_ttc, #fourn_ref, #tva_tx, #title_fourn_ref, #title_vat, #title_up_ttc, #title_up_ttc_currency").hide();
```

Comment:
You removed title_up_ht_currency but not title_up_ttc_currency ?
Is it done on purpose ?

Reply:
Yes, title_up_ttc_currency is non existent. DO you want me to remove it completely ?